build-system/server/app.js

```diff
@@ -1561,17 +1561,6 @@ app.get('/dist/diversions', (_req, res) => {
  * End Cache SW LOCALDEV section
  */
 
-/**
- * Web worker binary.
- */
-app.get('/dist/ww(.max)?.(m?js)', (req, res) => {
-  fs.promises.readFile(pc.cwd() + req.path).then((file) => {
-    res.setHeader('Content-Type', 'text/javascript');
-    res.setHeader('Access-Control-Allow-Origin', '*');
-    res.end(file);
-  });
-});
```

Comment:
Curious: Do test pages know how to fetch the correct version of this file?

Reply:
How can I check? I verified this worked by accessing a bind example file in the local server.

Reply:
https://github.com/ampproject/amphtml/issues/25024#issue-506203246 has repro step.

Reply:
I followed the repro steps and everything behaved properly! I had actually been using http://localhost:8000/examples/bind/basic.amp.html as my test case :).

Requests for `/dist/ww.js`, `/dist/ww.max.js`, `/dist/ww.max.js.map` all work even after a `gulp clean`. 
Requests for `/ww.js.map` fail, but I'm also unsure if anything requests that.

Reply:
Sweet! Glad we don't have to eager-build this.